internal/dvserver/api.go

```diff
@@ -2,18 +2,48 @@ package dvserver
 
 import (
 	"context"
+	"encoding/base64"
 	"fmt"
 
 	"go.uber.org/zap"
+	"google.golang.org/grpc/metadata"
 	"moul.io/depviz/v3/internal/dvcore"
 	"moul.io/depviz/v3/internal/dvmodel"
 	"moul.io/depviz/v3/internal/dvparser"
 	"moul.io/depviz/v3/internal/dvstore"
 )
 
+func getToken(ctx context.Context) (string, error) {
+	md, _ := metadata.FromIncomingContext(ctx)
+
+	var gitHubToken string
+	if md["authorization"] != nil {
+		// skip "Basic"
+		gitHubToken = md["authorization"][1][6:]
+		// prevent empty token (skip prefix)
+		if gitHubToken == base64.StdEncoding.EncodeToString([]byte("depviz:")) {
+			gitHubToken = ""
```

Comment:
```suggestion
			return gitHubToken, nil
```

could probably be rewritten this way for improved readability.

Reply:
👍